react/__tests__/CategoryItem.test.js

```diff
@@ -24,14 +23,16 @@ describe('CategoryItem component', () => {
       ],
     }
 
-    wrapper = render(<CategoryItem category={categoryMock} />)
+    wrapper = mount(<CategoryItem category={categoryMock} />)
   })
 
   it('should have the correct href', () => {
-    expect(wrapper.getByText('Category').href).toBe(`${location.href}#1`)
+    expect(wrapper.getDOMNode().querySelector('a').href).toBe(
+      `${location.href}#1`
+    )
   })
 
   it('should match snapshot', () => {
-    expect(wrapper.container).toMatchSnapshot()
+    expect(wrapper.getElement()).toMatchSnapshot()
```

Comment:
O que ta querendo ser testado/garantido aqui?

Reply:
é pra evitar uma mudança repentina indesejada no que o componente for renderizar. talvez não faça muito sentido isso ser testado agora no começo, porque o componente ainda ta sendo definido, e seja melhor deixar pra quando ele estiver mais completo. o que tu acha?

Reply:
Perfeito 👌 

Então, essa alinha ta errada pq o snapshot nao ta salvando o que foi renderizado:

![image](https://user-images.githubusercontent.com/284515/38884719-cedd6986-4246-11e8-8061-f6a830a82141.png)